docs/EN/Usage/Analyze-Graphs-Curves.md

```diff
@@ -0,0 +1,68 @@
+# Analyze graphs and curves
+
+You will find in this page some tick an trips to begin the analysis of the different curves and graphs.
+
+Always keep in mind that everyone is different. This page presents examples of curves and their interpretation to help you in the beginning, but you must make your own experience and learn to analyze your own curves according to your situation and what you did (meals, sports, stress ...) 
+
+
+## Prediction curves
+
+### Example of a standard meal with OpenAPS SMB
+
+You'll see in the pictures below the evolution of the prediction curves during the 4 hours after a meal
+
+![](C:\Users\Titi\StudioProjects\AndroidAPSdocs\docs\EN\images\4h_Meal_EN.jpg)
```

Comment:
links must be relative

Reply:
Thanks for the remark ... beginner error!